environment.yml

```diff
@@ -0,0 +1,56 @@
+channels:
+- conda-forge
+dependencies:
+- pip:
+  - flit
+  - bbknn
+  - scanpydoc>=0.7.3
+  - harmonypy
+  - magic-impute>=2.0
+  - cudf>=0.9
+  - cuml>=0.9
+  - cugraph>=0.9
+  - scanorama
+  - scrublet
+- python>=3.7
+- pip
+- anndata>=0.7.4
+- numpy>=1.17.0
+- matplotlib-base>=3.1.2
+- pandas>=0.21
+- scipy>=1.4
+- seaborn-split
+- h5py>=2.10.0
+- pytables
+- tqdm
+- scikit-learn>=0.22
+- statsmodels>=0.10.0rc2
+- patsy
+- networkx>=2.3
+- natsort
+- joblib
+- numba>=0.41.0
+- umap-learn>=0.3.10
+- packaging
+- sinfo
+- setuptools-scm
+- pytoml
+- black>=20.8b1
+- docutils
+- sphinx<4.2,>=4.1
+- sphinx_rtd_theme>=0.3.1
+- sphinx-autodoc-typehints
+- python-igraph
+- leidenalg
+- louvain!=0.6.2,>=0.6
+- scikit-misc>=0.1.3
+- pytest>=4.4
+- pytest-nunit
+- dask-core!=2.17.0
+- fsspec
+- zappy
+- zarr
+- profimp
+- flit-core
+name: scanpy
+
```

Comment:
Is this file necessary? It's in the base directory of scanpy in the r integration branch, but doesn't exist in the main fork (as far as I can tell). Just want to make sure we're not adding extra files accidentally